cypress/support/commands.js

```diff
@@ -26,9 +26,8 @@
 
 import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command'
 addMatchImageSnapshotCommand({
-  failureThreshold: 10.0,
+  failureThreshold: 0.10,
```

Comment:
`10` is quite big one to accept ! any speicific reason to back this ?
can we try with anything `<3` or `<5` ? WDYT

Reply:
10.0 is in real 10000% thus no 2e2 test could ever fail via diff... see [cypress-image-snapshot/issue/121](https://github.com/palmerhq/cypress-image-snapshot/issues/121)